spiff-arena-common/src/spiff_arena_common/coverage.py

```
@@ -39,19 +81,16 @@ def task_coverage(ctx):
     all = {}
     completed = {}
     missing = {}
-    for id, task_id in cov_tasks(ctx.test_cases):
+    embedded_subprocess_owners = _embedded_subprocess_owners(ctx.specs)
+    for id, task_id in cov_tasks(ctx.test_cases, embedded_subprocess_owners):
```

Ruff (0.15.12)

[error] Variable `id` is shadowing a Python builtin

(A001)